build.sbt

```diff
@@ -304,6 +310,22 @@ val jsdocs = project
   )
   .enablePlugins(ScalaJSPlugin, ScalaJSBundlerPlugin)
 
+val jswebsitedocs = project
+  .in(file("tests/websiteJs"))
+  .settings(
+    sharedSettings,
+    publish / skip := true,
+    scalaJSLinkerConfig ~= {
+      _.withModuleKind(ModuleKind.ESModule)
+    },
+    libraryDependencies ++= List(
+      "org.scala-js" %%% "scalajs-dom" % scalajsDom
+      // "com.raquo" %%% "laminar" % "0.17.0",
```

Comment:
I wanted to try and showcase it, but I think ended up concluding it would be a scala 3 thing for some reason. That is way outside the scope of what I'm trying to show here. I agree the comment should be removed.